isis/src/base/objs/EmbreeShapeModel/EmbreeShapeModel.truth

```diff
@@ -46,7 +46,7 @@ Embree shape model status:
 Model has intersection?  true
   Surface Point: ( 0.2908 ,  -0.0019 ,  0.0005 )
 Model has normal?  true
-  Surface Normal: ( 0.847198 ,  -0.194495 ,  0.494395 )
+  Surface Normal: ( 0.8471 ,  -0.1944 ,  0.4943 )
```

Comment:
Is this rounding a Mac vs Linux issue?

Reply:
Sort of, Embree only does floating point precision calculations and we sometimes see very small differences based on OS.